utilities/test_suite/HOST/runTests.py

```diff
@@ -145,37 +246,137 @@ def rpp_test_suite_parser_and_validator():
     print("QA mode can only run with a batch size of 3.")
     exit(0)
 
-# set the output folders and number of runs based on type of test (unit test / performance test)
 if(testType == 0):
     if qaMode:
         outFilePath = os.path.join(os.path.dirname(cwd), 'QA_RESULTS_HOST_' + timestamp)
     else:
         outFilePath = os.path.join(os.path.dirname(cwd), 'OUTPUT_IMAGES_HOST_' + timestamp)
     numRuns = 1
 elif(testType == 1):
-    if numRuns == 0:
-        numRuns = 100  #default numRuns for running performance tests
+    if "--num_runs" not in sys.argv:
+        numRuns = 1000 #default numRuns for running performance tests
```

Comment:
@sampath1117 Have we tested how default perf tests perform with 1000 runs? Some involved augmentations might take too long to run for 1000 runs. It should be comfortable for the user to check and quickly get a report, if they want an average over higher runs, that could be passed separately.